src/app/proposals/create/RemoveBuilderProposalForm.tsx

```diff
@@ -44,7 +45,7 @@ export const RemoveBuilderProposalForm: FC = () => {
   const proposalId = params?.get('proposalId') ?? ''
   const { isLoading: isVotingPowerLoading, canCreateProposal } = useVotingPower()
   const { setMessage } = useAlertContext()
-  const { onRemoveBuilderProposal, isPublishing, error } = useRemoveBuilderProposal()
+  const { onRemoveBuilderProposal, isPublishing } = useRemoveBuilderProposal()
```

Comment:
Should we handle `transactionError`?

Reply:
same as [here](https://github.com/RootstockCollective/dao-frontend/pull/336#discussion_r1830777121)